custom_components/pyscript/__init__.py

```diff
@@ -250,6 +266,55 @@ async def unload_scripts(global_ctx_only=None, unload_all=False):
         await GlobalContextMgr.delete(global_ctx_name)
 
 
+@bind_hass
+async def install_requirements(hass):
+    """Install missing requirements from requirements.txt."""
+    requirements_path = os.path.join(hass.config.path(FOLDER), "requirements.txt")
```

Comment:
See the discussion in the PR comments - it would be great to also check `requirements.txt` files in app and module subdirectories too.